site/content/en/docs/concepts/workload.md

```diff
@@ -95,6 +95,9 @@ For a `batch/v1.Job`, Kueue sets the priority of the Workload based on the
 [pod priority](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/)
 of the Job's pod template.
 
+Sometimes developers would like to control workload's priority without affecting pod's priority.  
```

Comment:
Prefer to make a change in line 92 that explains that there are two ways of setting the workload priority for a Job. Then use an unordered list to show the two options.